priv/templates/nimble_template/.github/wiki/Getting-Started.md.mix.eex

```diff
@@ -24,6 +24,16 @@
   mix test
   ```
 
+- Run all tests with `watch` option:
```

Comment:
same as this file is for the `mix variant`

Reply:
Good catch, finally we keep it, because we added `test.interactive` in the mix variant